.github/workflows/pr.yml

```diff
@@ -161,10 +161,13 @@ jobs:
           # Monterrey
           - macos-12
```

Comment:
Apparently macos-13 and macos-14 are available as Github runners. Should we switch to a newer macos here anyway?